src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -17639,6 +17639,39 @@ val:   -2
 ]]>)
 
         End Sub
+
+        <Fact>
+        Public Sub UnusedTuple()
```

Comment:
I ended up not adding user-defined conversions for VB, as we didn't see logic that looks for that in VB.